tests/integration/test_linspace.py

```diff
@@ -15,46 +15,277 @@
 
 import numpy as np
 import pytest
+from utils.generators import broadcasts_to, mk_seq_array
 
 import cunumeric as num
 
 
-def test_basic():
-    x = np.linspace(2.0, 3.0, num=5)
-    y = num.linspace(2.0, 3.0, num=5)
-    assert np.array_equal(x, y)
+def equivalent_shapes_gen(shape):
```

Comment:
please add a docstring explaining what this helper is for/does. I think this function removes leading singleton dimensions from shape tuples, one at a time, but please confirm. (It would be nice if there was an analogue to `np.squeeze` that worked on shapes rather than actual arrays, but alas there is not as far as I know)